dbt/adapters/athena/impl.py

```diff
@@ -124,6 +124,8 @@ class AthenaAdapter(SQLAdapter):
     AdapterSpecificConfigs = AthenaConfig
     Column = AthenaColumn
 
+    quote_character: str = '"'  # Presto quote character
```

Comment:
I think we already this kind of notion in AthenaColumn, would it be better from a code perspective to handle quote of a seed column in AthenaColumn, if feasible ?

Reply:
I've had a quick look and its not immediately obvious to me how that would be achieved. I think this quoting thing is only relevant for seeds so maybe my current implementation is all thats needed?

Happy to give it a go if you think its worth it though?

Reply:
Hum indeed, the base adapter has the methods quote_seed_column and quote and since they do not provide a way to pass quote char, your solutions is the most suitable 👌🏻